CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [0.2] - 2019-03-11
+### Added
+- Country Extension
+- 5 more entries in the `standalone` key
+- `categories-list` file with new categories 
```

Comment:
I would only list changes to the standard, rather than the structure of the documentation:

```suggestion
```